thirdeye/thirdeye-frontend/app/pods/manage/alert/explore/route.js

```diff
@@ -443,6 +395,98 @@ export default Route.extend({
     return RSVP.hash(promises);
   },
 
+  /**
+   * Performs the repetitive task of setting graph properties based on
+   * returned metric data and dimension data
+   * @method setGraphProperties
+   * @param {Object} metricData - returned metric timeseries data
+   * @param {String} exploreDimensions - string of metric dimensions
+   * @returns {RSVP promise}
+   */
+  setGraphProperties(metricData, exploreDimensions) {
+    const alertDimension = exploreDimensions ? exploreDimensions.split(',')[0] : '';
+    Object.assign(metricData, { color: metricDataColor });
```

Comment:
not very clear where `metricDataColor` is coming from, if it's a constant let's use uppercase `METRIC_DATA_COLOR`

Reply:
done